enhancements/update/clusterversion-history-pruning.md

```diff
@@ -0,0 +1,168 @@
+---
+title: clusterversion-history-pruning
+authors:
+  - "@jottofar"
+reviewers:
+  - "@LalatenduMohanty"
+  - "@wking"
+approvers:
+  - "@sdodson"
+api-approvers:
+  - N/A
+creation-date: 2022-06-10
+last-updated: 2022-06-10
+tracking-link:
+  - https://issues.redhat.com/browse/OTA-661
```

Comment:
nit: [`tracking-link`][1] docs are not super clear, but I'd expect a link to the Epic ( https://issues.redhat.com/browse/OTA-664 ) to be more useful for folks interested in this enhancement than a link to the ticket scoped to creating the enhancement.  But folks can get from one to the other, so not a big deal either way.

[1]: https://github.com/openshift/enhancements/blame/58894a670386b26ebc7699ad6904b6b939afac90/guidelines/enhancement_template.md#L13

Reply:
changed to epic